testsuite/tests/basic-modules/recursive_module_evaluation_errors.ml

```diff
@@ -17,19 +17,19 @@ Error: Cannot safely evaluate the definition of the following cycle
 Line 2, characters 10-20:
 2 | and B:sig val x: int end = struct let x = E.y end
               ^^^^^^^^^^
-  Module "B" defines an unsafe value, "x" .
+  Module "B" defines an unsafe value, "B.x" .
```

Comment:
The example that motivated the issue, with unsafe values within submodules, should be added somewhere in this test file.

Reply:
And the current changes look like a regression to me, the name of the module does not need to be repeated.